### DISCUSSION TITLE
[Code Coverage]Cannot find coverage file where cypress run with headless

### DISCUSSION THREAD

**Discussion by melonfarm2** [cypress-io/cypress#22268] [Plugins]
Hello.

I have a question about code coverage. I was applying code coverage by referring to the document in the link below. https://docs.cypress.io/guides/tooling/code-coverage

When running the test directly through the command `cypress open`, '.Folder `.nyc_output` and `coverage` are well created. However, if you test without a browser through `cypress run`, the above folder will not be created and the following message will appear.
```

  example to-do app
    √ displays two todo items by default (626ms)
    √ can add new todo items (1028ms)
    √ can check off an item as completed (222ms)
    with a checked task
      √ can filter for uncompleted tasks (314ms)
      √ can filter for completed tasks (312ms)
Cannot find coverage file ******.nyc_output\out.json
Skipping coverage report
      √ can delete all completed tasks (303ms)

```

How do we solve this?

Thank you.

**Comment by suresh-raja-nxtgen** [comment]
Hi @melonfarm2, I'm facing the same issue. Were you able to resolve the issue?

**Reply by melonfarm2** [reply]
I resolved this issue by adding the `--headed` option

**Reply by hamidmayeli** [reply]
Hi @melonfarm2 does it mean you did not run it in headless mode?
